examples/catalog/lib/custom_a11y_traversal.dart

```diff
@@ -92,13 +92,8 @@ class RowColumnTraversal extends StatelessWidget {
 /// set its traversal order.
 class SpinnerButton extends StatelessWidget {
   const SpinnerButton(
-      {Key key,
-      this.onPressed,
-      this.icon,
-      this.rowOrder,
-      this.columnOrder,
-      this.field,
-      this.increment}) : super(key: key);
+      { Key key, this.onPressed, this.icon, this.rowOrder, this.columnOrder, this.field, this.increment })
```

Comment:
the existing formatting here was terrible.
the new formatting is better but i think it'd be even better with a trailing comma and the resulting formatting